WolvenKit.App/ViewModels/Exporters/ExportViewModel.cs

```diff
@@ -178,6 +178,26 @@ private async Task<bool> ExportSingleAsync(ExportableItemViewModel item, FileSys
 
         return await _importExportHelper.Export(fi, settings, new DirectoryInfo(projectArchive.Project.ModDirectory), new DirectoryInfo(projectArchive.Project.RawDirectory));
     }
+    
+    /// <summary>
+    /// Check if the file should be included in the export tool's view. Currently excluded:
+    /// - .ent files (they should be exported via wscript instead)
+    /// - .app files (this isn't handled)
+    /// - .inkatlas files
+    /// </summary>
+    /// <param name="filePath">The file path as string</param>
+    /// <returns>true if this file should not be shown in the import browser</returns>
+    private static bool ShouldFileBeFiltered(string filePath) =>
+        Path.GetExtension(filePath) switch
+        {
+            ".inkatlas" => // we're not handling this at the moment
+                true,
+            ".ent" => // .ent files - export them via script
```

Comment:
Also do this via `CanExport` then? Or is the import of it needed?

Reply:
OK, I'm gonk :D I'll look up CanExport!